pkg/mover/chart.go

```diff
@@ -139,6 +141,7 @@ type ChartMover struct {
 	chart                     *chart.Chart
 	logger                    Logger
 	retries                   uint
+	bundle                    *intermediateBundle
```

Comment:
can we call it intermediateBundle?

Reply:
ok, will fix.

I do not love long names, and in a typed language like go, they are even less needed, as this is clearly an `intermediateBundle`, says so in the type. But will rename as requested.

Reply:
renamed